src/mono/mono/utils/mono-counters.c

```diff
@@ -411,6 +425,7 @@ initialize_system_counters (void)
 	register_internal ("User Time", SYSCOUNTER_TIME, (gpointer) &user_time, sizeof (gint64));
 	register_internal ("System Time", SYSCOUNTER_TIME, (gpointer) &system_time, sizeof (gint64));
 	register_internal ("Total Time", SYSCOUNTER_TIME, (gpointer) &total_time, sizeof (gint64));
+	register_internal ("Real Time", SYSCOUNTER_TIME, (gpointer) &real_time, sizeof (guint64));
```

Comment:
nit: "Wall Clock Time" ?

Reply:
It uses `CLOCK_MONOTONIC` rather than `CLOCK_REALTIME` or `CLOCK_BOOTTIME `, so I'm somewhat hesitant on that.